capstone/src/main/java/com/google/univiz/scorecard/URLProviderImpl.java

```diff
@@ -14,8 +15,16 @@
 import javax.inject.Inject;
 
 final class URLProviderImpl implements URLProvider {
+  private static final String FRONT_URL =
+      "https://api.data.gov/ed/collegescorecard/v1/schools.json?";
+  private static final String QUERY_TYPE_NAME = "school.name=";
+  private static final String QUERY_TYPE_ID = "id=";
+  private static final String PER_PAGE = "&per_page=";
+  private static final String FIELDS_PARAM = "&fields=";
+  private static final String SUGGESTION_PER_PAGE = "20";
   private final UnivizConfig univizConfig;
   private final Set<String> fields;
```

Comment:
This variable name is now ambiguous. Rename to something more specific. Like resourceDataFields, dataFields, resourceFields, etc.

Reply:
changed it to collegeDataFields